modules/java_api/src/test/java/org/intel/openvino/CoreTests.java

```diff
@@ -1,8 +1,7 @@
-package tests;
+package org.intel.openvino;
```

Comment:
@rajatkrishna, This breaks Android build: https://github.com/dkurt/openvino_java/actions/runs/5219539621/jobs/9441828252?pr=22

steps to reproduce: https://github.com/dkurt/openvino_java/blob/test_java_refactoring/.github/workflows/android.yaml

```
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/TensorTests.java:3: error: package org.junit does not exist
import static org.junit.Assert.*;
                       ^
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/TensorTests.java:5: error: package org.junit does not exist
import org.junit.Test;
                ^
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/OVTest.java:3: error: package org.junit does not exist
import org.junit.Ignore;
                ^
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/OVTest.java:4: error: package org.junit does not exist
import org.junit.Rule;
                ^
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/OVTest.java:5: error: package org.junit.rules does not exist
import org.junit.rules.TestWatcher;
                      ^
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/OVTest.java:6: error: package org.junit.runner does not exist
import org.junit.runner.Description;
                       ^
/home/runner/work/openvino_java/openvino_java/openvino_contrib/modules/java_api/src/test/java/org/intel/openvino/OVTest.java:10: error: cannot find symbol
@Ignore
 ^
  symbol: class Ignore
```

Reply:
It appears that the compilation errors are occurring due to missing JUnit dependencies. Due to these changes, gradle detects the java tests in the `src/test/java` folder and attempts to compile them. 

One solution would be to disable the compilation and execution of tests through gradle using the `-x test` flag when building for Android: https://github.com/dkurt/openvino_java/pull/24

Please let me know if you have questions or if there are alternative solutions you'd like to explore. Thank you